pkgs/lib.nix

```diff
@@ -83,4 +83,15 @@ let self = {
     map = [ map ];
     version = 3;
   };
+
+  address = addr:
```

Comment:
Took me a moment to understand that this is for clients (I missed the commit description). Perhaps add a comment explaining what this is for?